.github/workflows/style.yml

```diff
@@ -55,4 +55,4 @@ jobs:
           command: doc
           args: --no-deps --all-features
         env:
-          RUSTDOCFLAGS: --cfg docsrs
+          RUSTDOCFLAGS: --cfg docsrs -Dwarnings
```

Comment:
I'm not a fan of denying warnings, even in CI because new warnings will _always_ accumulate over time simply as they get added to rustc/rustdoc/clippy. This is why we have GitHub annotations for PRs, which indicate the warnings without blocking the PR.

Reply:
Done!